src/Dialect/ONNX/ONNXOps/Tensor/ConstantOfShape.cpp

```diff
@@ -59,8 +59,9 @@ LogicalResult ONNXConstantOfShapeOp::verify() {
   auto inputShape = input.getType().cast<RankedTensorType>().getShape();
   if (inputShape.size() != 1)
     return emitOpError("Input tensor must be a 1D tensor");
+
   if (ShapedType::isDynamic(inputShape[0]))
-    return emitOpError("Input tensor must have static shape");
```

Comment:
it seems wrong for verify() to require the input shape to be static - it could start dynamic and become static during shape inference

Reply:
It makes sense.